prow/config.yaml

```diff
@@ -120,7 +120,16 @@ branch-protection:
             contexts:
             - deploy/netlify
         cni:
+          required_pull_request_reviews: *tide_withcodeowners_reviews
```

Comment:
```suggestion

```

Now that #1666 has merged , the `tide_withcodeowners_reviews` and `tide_withcodeowners_restrictions` anchors are no longer defined.

Reply:
Otherwise, there may be unintentional conflicts or reversions.